src/libs/focusWithDelay.ts

```diff
@@ -1,15 +1,13 @@
-import {InteractionManager} from 'react-native';
+import {InteractionManager, TextInput} from 'react-native';
 import ComposerFocusManager from './ComposerFocusManager';
 
 /**
  * Create a function that focuses a text input.
- * @param {Object} textInput the text input to focus
- * @returns {Function} a function that focuses the text input with a configurable delay
  */
-function focusWithDelay(textInput) {
+function focusWithDelay(textInput: TextInput | null): (shouldDelay?: boolean) => void {
```

Comment:
Could you extract `(shouldDelay?: boolean) => void` in a separate type and use it here?